submissions/idiagheOsaigbovo/html/htmlform.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
```

Comment:
Good attempt Osas. Please fix all indentations on this document. Refer to [Styleguide](https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp19/blob/master/STYLEGUIDE.md)

Reply:
okay i will do just that.